sdk/extensions/Microsoft.Extensions.Azure/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.1.0-beta.2 (Unreleased)
+## 1.1.0-beta.2 (2021-02-09)
```

Comment:
Was this intended?

Reply:
Yeah, I'll ship that one as well decided to kill 2 birds with one stone.